pom.xml

```diff
@@ -418,6 +418,18 @@
       <version>${assertj.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.assertj</groupId>
+      <artifactId>assertj-core</artifactId>
+      <version>3.9.1</version>
+      <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.assertj</groupId>
+      <artifactId>assertj-core</artifactId>
+      <version>3.9.1</version>
+      <scope>compile</scope>
+    </dependency>
```

Comment:
2mal? 

Vielleicht sollten wir doch die Assertions in den test Folder verschieben, dann kann der Scope auf test gesetzt werden.